kalbarri-workshop.md

```diff
@@ -421,7 +424,7 @@ Map.addLayer(ps4Img, ps4VisParams, 'PS4 Image');
 
 ### ImageCollections
 
-An `ImageCollection` in Google Earth Engine is a stack of `Image`s; it provides a structure to group together and organise related `Image`s. For example, you could create an `ImageCollection` that stores all `Image`s captured by a satellite such as Landsat, Sentinel-2, or Planet. 
+An `ImageCollection` in Google Earth Engine is a stack of `Image`s; it provides a structure to group together and organise related `Image`s. For example, you could create an `ImageCollection` that stores all the Kalbarri `Image`s captured by the Planet satellite over a 6 month period. 
```

Comment:
Timeseries analogy would relate better to this lab than other satellite products